src/main/java/seedu/address/logic/Messages.java

```diff
@@ -38,9 +38,9 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
     public static String format(Person person) {
         final StringBuilder builder = new StringBuilder();
 
-        builder.append(person.getStudentId())
```

Comment:
Is there any particular reason why you swapped these lines? I think this could possibly affect many of the test cases (don't quote me on that though)

Reply:
Yea, there is. Because if I dont swap the two lines, then the message shown when command is successful will be wrong.
For example: the add command
The message shown when command is successful:
(before swapping):
**New person added: A1234567U; StudentId: Ady**; Email: e0123456@u.nus.edu; Grade: B; Groups: 
(after swapping):
**New person added: Ady; StudentId: A1234567U**; Email: e0123456@u.nus.edu; Grade: B; Groups:

Reply:
Ok, LGTM then.